### PR TITLE
Project: add ci_config_path property

### PR DESCRIPTION
**Pull request by stapelberg**

### CODE REVIEW

File: projects.go [hunk 1]
```diff
@@ -83,8 +83,9 @@ type Project struct {
 		GroupName        string `json:"group_name"`
 		GroupAccessLevel int    `json:"group_access_level"`
 	} `json:"shared_with_groups"`
-	Statistics *ProjectStatistics `json:"statistics"`
-	Links      *Links             `json:"_links,omitempty"`
+	Statistics   *ProjectStatistics `json:"statistics"`
+	Links        *Links             `json:"_links,omitempty"`
+	CiConfigPath *string            `json:"ci_config_path"`
```
Comment (zaquestion) [inline projects.go]: Consider capitalizing the `I` as in `CIConfigPath`. Go likes capitalized acronyms 
Ref: https://github.com/golang/go/wiki/CodeReviewComments#initialisms

**Comment by svanharmelen** [comment]
Thanks @stapelberg! If you could just update the field name like @zaquestion suggested, I will merge the PR right after that 👍

File: projects.go (hunk 1 above)
Comment (stapelberg) [reply projects.go]: I copied this from the already-existing https://github.com/xanzy/go-gitlab/blob/062bdff086ddaf6a8de7335bcd57bf7d468c4967/projects.go#L353 — shouldn’t we keep the two consistent?

Comment (svanharmelen) [reply projects.go]: Ah... Guess that one slipped through 😞  But I rather fix the other one in that case...

**Comment by stapelberg** [comment]
Done.

**Comment by svanharmelen** [comment]
Thanks again @stapelberg! LGTM!

**Pull request merged**